### PR TITLE
Internal: Fixing breakpoints in core

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/playwright/assets/breakpoints.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default class {
 			await this.page.goto( `/wp-admin/post.php?post=${ experimentPostId }&action=elementor` );
 		} else {
 			await this.page.reload();
-
+			await this.page.reload();
```
Comment (mykytamurzin) [inline tests/playwright/assets/breakpoints.ts:39]: double page reload?

Comment (dmaziar) [reply tests/playwright/assets/breakpoints.ts:39]: yes. After the first one, left panel is hanging. Looks like second one should help.

**Review by louiswol94 — CHANGES_REQUESTED** [review]
Is it possible to rather identify the root cause? :-)

**Comment by dmaziar** [comment]
> Is it possible to rather identify the root cause? :-)

The root cause is 
```
e.O is not a function
TypeError: e.O is not a function
    at http://localhost:8888/wp-content/plugins/build/assets/js/frontend.min.js?ver=3.18.0:2:40340
    at webpackJsonpCallback (http://localhost:8888/wp-content/plugins/build/assets/js/styleguide-app-initiator.min.js?ver=3.18.0:3:12265)
    at webpackJsonpCallback (http://localhost:8888/wp-content/plugins/build/assets/js/webpack.runtime.min.js?ver=3.18.0:2:4680)
    at http://localhost:8888/wp-content/plugins/build/assets/js/frontend.min.js?ver=3.18.0:2:74
```
I see this error on hosting website frequently

**Pull request closed without merging**